src/libcoretest/mem.rs

```diff
@@ -55,7 +56,8 @@ fn align_of_basic() {
 #[cfg(any(target_arch = "x86",
           target_arch = "arm",
           target_arch = "mips",
-          target_arch = "mipsel"))]
+          target_arch = "mipsel",
+          target_arch = "powerpc"))]
```

Comment:
Hm, to stem the flow of these additional `target_arch` settings, can you change this to be conditional on `target_pointer_width`?